script/hassfest/requirements.py

```diff
@@ -134,11 +173,23 @@ def install_requirements(integration: Integration, requirements: Set[str]) -> bo
 
     Return True if successful.
     """
+    global PIPDEPTREE_CACHE
+
+    ensure_cache()
+
     for req in requirements:
-        try:
-            is_installed = pkg_util.is_installed(req)
-        except ValueError:
-            is_installed = False
+        is_installed = False
+
+        if "==" in req:
+            pkg, ver = req.split(" ")[-1].split("==")
```

Comment:
Is the regex search we have below a lot slower than this?

Reply:
Oh I didn't see that. Just went to add cache support without really looking at rest of function